### DISCUSSION TITLE
Mount as non removable media

### DISCUSSION THREAD

**Discussion by SNisin** [openzfsonwindows/openzfs#349] [Q&A] [answered]
I noticed that the ZFS pool and filesystems are mounted on Windows as removable media. Why is that so? Is there an option to mount them as standard local disks instead? I'd like to use Backblaze Personal for backup, but they don't support removable media.

**Comment by guenther-alka** [comment]
I suppose there are at least two reasons:

1. A ZFS pool can be dynamically mounted with different mountpoints like c:\pool x:\ or any other location
2. A modification of some ZFS properties like driveletter, nbmand or readonly require a unmount/mount cycle.

Only a removeable media can do.

**Comment by SNisin** [comment]
I found now [in the source code](https://github.com/openzfsonwindows/openzfs/blob/555691146c15e73157c401850fdebb0b141e4e52/module/os/windows/zfs/zfs_vnops_windows_mount.c#L738). According to the comments in the code, it seems like the only reason for it is to prevent $recycle.bin. The variable zfs_disable_removablemedia is set to 0 and it looks like there is no way to change that without recompilling.

**Comment by lundman** [comment, marked as answer]
400fc2e

**Reply by SNisin** [reply]
Thank you very much

**Comment by guenther-alka** [comment]
Suggestion:  Add a wiki table of the say top 20 most common Open-ZFS registry entries with their suggested default values and possible values to safely modify and reset settings?
